ui/README.md

```diff
@@ -0,0 +1,23 @@
+# Kubecost Open Source UI
```

Comment:
This documentation LGTM, and we can plan to link from the repo's primary readme in a future release too. 

I may just add the note at the top of this section about our recommended install path! 
https://github.com/kubecost/cost-model/blob/develop/kubecost-exporter.md#installing

Reply:
Definitely agree about highlighting the regular install path.